environment-lock.yml

```diff
@@ -87,3 +85,4 @@ dependencies:
   - zlib=1.2.12=h166bdaf_1
   - pip:
     - remote-pdb==2.1.0
+prefix: /opt/miniconda/envs/slackbotr
```

Comment:
The prefix shouldn't be present. The new invoke task will ensure that's the case. Did you use it and this still happened?

Reply:
i did use it and this still happened